veridaOne/profile/v0.1.0/schema.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://common.schemas.verida.io/veridaOne/profile/v0.1.0/schema.json",
+  "name": "https://common.schemas.verida.io/veridaOne/profile/v0.1.0/schema.json",
+  "title": "Verida One Profile",
+  "titlePlural": "Verida One Profiles",
+  "description": "Configuration of a profile on Verida One",
+  "type": "object",
+  "database": {
```

Comment:
Also add `appearance` and `layouts`

See https://github.com/verida/schemas-common/blob/b9dc8ebf3d306f21893fb6784dd74416b5094f59/social/following/v0.1.0/schema.json#L8

Reply:
All the schema's properties are arrays, does it work in layouts?